supersetapiclient/base.py

```diff
@@ -271,13 +271,18 @@ def find(self, **kwargs):
                         "opr": "eq",
                         "value": v
                     } for k, v in kwargs.items()
-                ]
+                ],
+                "page_size": page_size,
+                "page": page
             }
             params = {
                 "q": json.dumps(query)
             }
         else:
-            params = {}
+            query = {"page_size": page_size, "page": page}
+            params = {
+                "q": json.dumps(query)
+            }
```

Comment:
I think you should factorise that and initialise the query before the if... else (line 266)

```python
       query = {
                "page_size": page_size,
                "page": page
       }

        if kwargs != {}:
        
        ...
        
             query = query.update({...})
             params = ...
```